scripts/log_api_gateway.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import boto3
+import json
+
+# This script should only be called by Terraform.
+# Do NOT run manually.
+
+
+# Converted to a string that expresses the structure of API log entries
+# For more info see https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html
+JSON_LOG_FORMAT = {
+    "requestId": "$context.requestId",
+    "ip": "$context.identity.sourceIp",
+    "caller": "$context.identity.caller",
+    "user": "$context.identity.user",
+    "requestTime": "$context.requestTime",
+    "httpMethod": "$context.httpMethod",
+    "resourcePath": "$context.resourcePath",
+    "status": "$context.status",
+    "protocol": "$context.protocol",
+    "responseLength": "$context.responseLength"
+}
+
+client = boto3.client('apigateway')
```

Comment:
Avoid side effects at module scope. It's fine to have a throw-away script have side effects at module scope but then it would make sense to have the if `__name__ == __main__` stuff at the bottom. That conditional implies that the script could also be imported. I think it's generally safe practice to always do `if __name__ … ` and never have side-effects at module scope.

Reply:
Good point. I moved it out to avoid redundant calls, but I suppose it could just indirectly add more. Instead I'll put it in the `add_field` function again.